apollo/src/graphql/model/modelQueries.ts

```diff
@@ -1,31 +1,50 @@
 import { MLModel, ObjectionMLModel } from './model.js';
 import { builder } from '../../builder.js';
+import { MLModelConnection } from './modelConnection.js';
 
 builder.queryFields((t) => ({
     listMLModels: t.field({
-        type: [MLModel],
+        type: MLModelConnection,
         args: {
             modelName: t.arg.string(),
-            limit: t.arg({
+            first: t.arg({
                 type: 'Limit',
                 defaultValue: 10,
+                required: true,
             }),
-            offset: t.arg.int({
-                defaultValue: 10,
-            }),
+            after: t.arg.string(),
         },
-        async resolve(_root, args, _ctx) {
+        async resolve(_root, { modelName, first, after }, _ctx) {
```

Comment:
any reason to pass the individual arguments instead of the `arg` object? Just curious, I don't have an opinion on it either way

Reply:
No specific reason. I can change back to `arg` so that it's consistent w/ the other methods.

Reply:
all hunky dory either way, was just curious!